devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

```diff
@@ -149,7 +155,7 @@ def levelEventHandler(currentLevel) {
 
 	log.debug "levelEventHandle - currentLevel: ${currentLevel} lastLevel: ${lastLevel}"
 
-	if (lastLevel == "undefined" || currentLevel == lastLevel) { //Ignore invalid reports
```

Comment:
@KKlimczukS  @JeremeLau I don't think that we want to remove this.

Reply:
@JeremeLau Please add the `|| currentLevel == lastLevel` back in this line. If there is a reason that you are removing it, please explain here. Thanks